### PR TITLE
Supports mongodb persistence driver

### PR DESCRIPTION
**Pull request by ne0h12**

### CODE REVIEW

File: Resources/config/services.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ services:
       class: 'Enqueue\ElasticaBundle\Doctrine\Queue\SyncIndexWithObjectChangeProcessor'
       public: true
       arguments:
-          - '@doctrine'
+          - null
```
Comment (makasim) [inline Resources/config/services.yml]: could you please add a comment that the right manager will be injected in the extension

File: DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -19,6 +19,9 @@ public function getConfigTreeBuilder()
                 ->scalarNode('context')->defaultValue('enqueue.transport.context')->cannotBeEmpty()->end()
                 ->arrayNode('doctrine')
                     ->children()
+                        ->scalarNode('driver')->isRequired()->cannotBeEmpty()
```
Comment (makasim) [inline DependencyInjection/Configuration.php]: set ORM as default value otherwise it is a BC break

**Comment by ne0h12** [comment]
@makasim, thanks for review, I'm fixed this.

**Comment by makasim** [comment]
Thank you for the contribution.

**Pull request merged**